pom.xml

```diff
@@ -731,13 +731,14 @@
         <plugin>
           <groupId>com.github.spotbugs</groupId>
           <artifactId>spotbugs-maven-plugin</artifactId>
-          <version>4.7.3.6</version>
+          <version>4.9.3.0</version>
           <configuration>
             <xmlOutput>true</xmlOutput>
             <effort>Max</effort>
             <failOnError>true</failOnError>
             <includeTests>true</includeTests>
             <maxRank>16</maxRank>
+            <omitVisitors>ConstructorThrow,SharedVariableAtomicityDetector</omitVisitors>
```

Comment:
Why are these omitted?

Reply:
NVM, I see that this was explained in the overall comment for this PR.
For SharedVariableAtomicityDetector, may need to open a follow on issue about this. Should probably just suppress when needed as this may be hiding some bugs in the current code

Reply:
> For SharedVariableAtomicityDetector, may need to open a follow on issue about this. 

Agreed, I was going to do that for main. Each instance likely needs to be evaluated. Looking at https://github.com/spotbugs/spotbugs/issues/3390, we can probably just mark our not thread-safe classes with an annotation. I'm not sure that there is a release yet with this change in it.